styles.css

```diff
@@ -94,19 +94,35 @@ li.century {
     border: 2px solid black;
     box-shadow: 5px 3px 3px #3f2a14;
     background-color: white;
-    height: 40vh;
+    min-height: 29vw;
```

Comment:
Using vw rather than vh for height will make the elements a consistent size whether the screen is landscape or portrait